### PR TITLE
Cyborg Pointing and Deselect/Unequip Module Hotkeys

### PR DESCRIPTION
**Pull request by Krausus**
This PR gives cyborgs the ability to point with Middle Click (module cycling is now on Ctrl+Middle Click), and makes the existing Drop Item and Toggle Throw hotkeys deselect or unequip, respectively, the current module.

The current lack of cyborg pointing is... odd. As far as I can tell, when middle-click-to-point was implemented [in this commit](https://github.com/ParadiseSS13/Paradise/commit/98a99ebd1a227ddffbc0fe48d573097d318c3088), the fact that cyborgs had a custom `MiddleClickOn` wasn't considered, and nobody got around to fixing it. I can think of no reason for them *not* to be able to point, so now they can do so, with the same click carbons use. Module cycling has been moved to Ctrl+Middle Click, in case anyone still wants to cycle with the mouse.

* Cyborgs can now point with Middle Click.
  * Drones cannot point, as they are not intended to communicate outside of dronespeak.
  * The existing module cycling function of middle clicks is now done with Ctrl+Middle Click.
* For cyborgs, the Drop Item hotkey will now deselect the currently-selected cyborg module.
  * Handy for switching over to computer/machine interaction without having to click in the UI.
  * Previously, it simply emitted a "This mob type cannot drop items" error.
    * Even if you were using the magnetic gripper, which has its own special drop verb.
* For cyborgs, the Toggle Throw hotkey will now unequip the currently-selected cyborg module, and cycle to the next one (if any).
  * Handy for quickly swapping out modules without having to futz with the UI.
  * Previously, it simply emitted a "This mob type cannot throw items" error.
* The Hotkeys Help listing has been updated to include the two new module management hotkeys.
  * It seems like the whole listing could use a look-over to give it more user-friendly wording.
* Unequipping a cyborg module will now properly update the list of modules.
  * This was bugging me during testing, so I fixed it, because why not.

I've tested out these changes and everything looks good, so as long as nobody has objections, this should be good to merge once the waiting period is over.

### CODE REVIEW

**Comment by ZomgPonies** [comment]
Ideally I think Id' like to you reverse the behavior so it's standardized across all character types. MiddleClick to point, Ctrl-MiddleClick to cycle modules.

File: code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -43,6 +43,14 @@
 	if(iscarbon(usr))
 		var/mob/living/carbon/C = usr
 		C.toggle_throw_mode()
+	else if(isrobot(usr))
+		var/mob/living/silicon/robot/R = usr
+		var/module = R.get_selected_module()
+		if(!module)
+			usr << "\red You have no module selected."
```
Comment (SirBloodBuske) [inline code/modules/mob/mob_movement.dm]: "<span class='warning'>You have no module selected.</span>"

Wouldn't this be better?

**Comment by Krausus** [comment]
@ZomgPonies 
I considered that, but cyborgs have had unique middle-mouse module cycling all this time, and I'd feel bad if a long-time cyborg player tried to cycle modules at a pivotal moment but just pointed menacingly instead (even though it would also be hilarious). Also, same-click-but-with-ctrl is already how cyborg examining works, so it wouldn't be the first exception to the norm.

I suppose cyborgs probably *would* learn pretty quickly after some unintentional pointing, though. I can swap them if you feel the consistency is important.

File: code/modules/mob/mob_movement.dm (hunk 1 above)
Comment (Krausus) [inline code/modules/mob/mob_movement.dm]: The red error messages are consistent with other ones produced by those hotkeys.

**Comment by ZomgPonies** [comment]
I do think consistency is important when it comes to input. People complained back when I forced point to be MMB, but everyone got used to it. This is more of the same.

**Comment by Spacemanspark** [comment]
Pretty sure it already is.
As for this PR, :+1:.

**Comment by Krausus** [comment]
It is done.

Also, yes, cyborgs can already cycle modules with the same hotkey humans use to swap hands. Nobody *really* needed the middle-mouse module cycling, but they can still do it with ctrl if they wish.

**Pull request merged**